README.md

```diff
@@ -127,9 +126,12 @@ memphisConnection.close();
 
 ### Creating a Station
 
+_If a station already exists, the returned station will be returned, the new configuration will not apply_
```

Comment:
_If a station already exists nothing happens, the new configuration will not be applied_